### DISCUSSION TITLE
Jenkins build failure

### DISCUSSION THREAD

**Discussion by haseeb-abdul9** [learndevopsonline/learndevopsonline#1862] [Q&A]
Trying to run terraform-roboshop over Jenkins but the provisioners failing 
 jenkins server : 3.239.227.41 | 172.31.8.215 jenkins user & pass : admin | admin123
workstation server : 3.231.216.239 | 172.31.7.30

Repositories 
https://github.com/haseeb-abdul9/roboshop_terraform.git

https://github.com/haseeb-abdul9/roboshop_shell.git

**Comment by chaithanya1812** [comment]
# This was your code pls go through once, you can understand better.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/35e8fad9-a53c-42ef-89a7-eec686b5457b)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/5e6fa634-318a-4784-9b1e-8c13eb2385ee)

**Comment by chaithanya1812** [comment]
# Answer 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/079ca18b-dc71-42c7-bf42-fb7fdfdfa89f)

**Reply by haseeb-abdul9** [reply]
Thank you so much for taking the time
